web/src/beta/components/Resizable/README.md

```diff
@@ -0,0 +1,5 @@
+# Resizable
```

Comment:
copied from classic and edited gutter width

Reply:
These kind of READMEs can be not copied from classic. They are from a lonnggggggg time ago and we don't want them anymore 👍🏻

Reply:
thank you! I will delete and move my comment to the component file!